elasticdl/python/master/master.py

```diff
@@ -236,6 +236,11 @@ def run(self):
                             InstanceManagerStatus.FINISHED
                         )
                     break
+                if self.instance_manager.all_workers_completed:
```

Comment:
What will happen if half workers completed, the other half failed?

Reply:
We can fix the logic. The worker exits if completed or failed and the master exits if all workers exit.